fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -187,6 +184,14 @@ def self.run(options)
         end
       end
 
+      def self.puts_download_url(body)
+        version_url = body['public_url']
+        version_url = body['public_url'] + '/app_versions/' + body['config_url'].split('/').last if body['config_url']
```

Comment:
Not super handy on ruby... but in this case the config_url is there always.
In case is empty should split break? (if no '/' are found?)
I asume last will work even if 1 element is present